### DISCUSSION TITLE
Cant get angularGridInstance

### DISCUSSION THREAD

**Discussion by DiegoTrapiello** [ghiscoding/Angular-Slickgrid#885] [Q&A] [answered]
I'm not sure if this is the correct place to ask this so I apologize in advance.
I'm currently developing an app and using Slickgrid. I want to add a Datagrid item but keep getting this error:

Error: src/app/tabla/tabla.component.html:7:102 - error TS2339: Property 'detail' does not exist on type 'Event'.
   [gridOptions]="gridOptions" [dataset]="tabla.rows" (onAngularGridCreated)="angularGridReady($event.detail)">

I saw your Add demo and also this wiki page: https://github.com/ghiscoding/Angular-Slickgrid/wiki/Grid-&-DataView-Events#example-with-event-dispatch-sgonx and still dont know what I'm doing wrong.

**Comment by ghiscoding** [comment, marked as answer]
that's Angular-Language-Service in your editor that is being too strict and simply doesn't like Custom Event, see Discussion https://github.com/ghiscoding/Angular-Slickgrid/discussions/815 and Stack Overflow [Cannot use onAngularGridCreated emitter](https://stackoverflow.com/questions/71156193/cannot-use-onangulargridcreated-emitter/71245004#71245004)

**Reply by DiegoTrapiello** [reply]
Thanks, that fixed it!

**Reply by ghiscoding** [reply]
Added a section in the Wiki you were looking at to help others as well [Error thrown in the IDE when using `strictTemplates`](/ghiscoding/Angular-Slickgrid/wiki/Grid-&-DataView-Events#error-thrown-in-the-ide-when-using-stricttemplates)
